src/main/kotlin/com/yedongsoon/example_project/domain/schedule/Schedule.kt

```diff
@@ -0,0 +1,52 @@
+package com.yedongsoon.example_project.domain.schedule
+
+import com.yedongsoon.example_project.application.schedule.model.ScheduleCreateCommand
+import jakarta.persistence.*
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+@Entity
+@Table(name="schedule")
+class Schedule (
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name="schedule_no")
+    val scheduleNo : Int = 0,
+
+    @Column(name="account_no")
+    val accountNo : Int = 0,
+
+    @Column(name="schedule_name")
+    var scheduleName : String = "",
+
+    @Column(name="schedule_location")
+    var scheduleLocation : String = "",
+
+    @Column(name="schedule_with")
+    var scheduleWith : String = "",
+
+    @Column(name="group_gender_type")
+    var groupGenderType : String = "",
+
+    @Column(name="schedule_start_at")
+    var scheduleStartAt: LocalDateTime = LocalDateTime.now(),
+
+    @Column(name="schedule_end_at")
+    var scheduleEndAt: LocalDateTime = LocalDateTime.now(),
+
+    @Column(name="schedule_at")
+    var scheduleAt: LocalDate = LocalDate.now()
+){
+    companion object {
+        // 일정 등록 (command -> Schedule)
+        fun create(command : ScheduleCreateCommand) = Schedule(
```

Comment:
```suggestion
        fun create(command : ScheduleCreateCommand) = Schedule(
           accountNo = command.accountNo,
```
command 객체에서 엔티티로 넘어올 때 accountNo를 안넘기고 있었어용

Reply:
수정했습니다 👍🏻